src/DatabaseMigrationsTester/Program.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+
+namespace DatabaseMigrationsTester
+{
+    using System.Threading;
+
+    class Program
+    {
+        static int Main(string[] args)
+        {
+            var command = (Command)Enum.Parse(typeof(Command), args[0]);
```

Comment:
What is this for? Manual testing?

Reply:
This project is used by the `DatabaseMigrationsTest` because the migrations process needs to  spawn a new process and expects that process to behave in a certain way. This little console app simulates the behavior of a process running an embedded RavenDB>.